indexer/services/ender/src/scripts/handlers/dydx_stateful_order_handler.sql

```diff
@@ -22,9 +22,40 @@ DECLARE
     order_record orders%ROWTYPE;
     subaccount_record subaccounts%ROWTYPE;
 BEGIN
+    /* For order replacement, remove old order first and don't return immediately */
+    IF event_data->'orderReplacement' IS NOT NULL THEN
+        order_id = event_data->'orderReplacement'->'oldOrderId';
+        order_record."status" = 'CANCELED';
```

Comment:
I wonder if we want to store somewhere that this order was replaced & be able to trace it to its replacement order. Wdyt?

Reply:
I think might be worth doing in a future PR. I'm not sure what that change would affect downstream and don't want to break existing behavior